Automattic/studio
# Studio: Fix stop server error

Fixes: https://github.com/Automattic/dotcom-forge/issues/7245

## Proposed Changes

This PR fixes a bug when a user start/stops the server multiple times, child process exits with error 15

## Testing Instructions
1. Start Studio, and start/stop a site in the studio multiple times.
2. Ensure that an error is logged in from time to time, but there is no unhandled errors.



## Pre-merge Checklist
